libc-bottom-half/sources/__wasilibc_real.c

```diff
@@ -661,8 +661,8 @@ __wasi_errno_t __wasi_sock_shutdown(
 
 #ifdef _REENTRANT
 int32_t __imported_wasi_thread_spawn(int32_t arg0) __attribute__((
-    __import_module__("wasi"),
-    __import_name__("thread-spawn")
+    __import_module__("wasi_snapshot_preview1"),
```

Comment:
I believe this is incorrect at the moment, though probably not the biggest deal: (a) wasi-threads was not included in preview1 and (b) the spec (and implementations like Wasmtime) use the other name. I do prefer the consistency, though. I'd be ok adding this in for consistency reasons if you want to make all the necessary changes in the applicable places (Wasmtime, WAMR, spec).

Reply:
I don't have a strong opinion on that, so I'm reverting this change.